pkgs/os-specific/linux/lxc/default.nix

```diff
@@ -67,6 +67,22 @@ stdenv.mkDerivation rec {
 
   postInstall = ''
     wrapPythonPrograms
+
```

Comment:
I'm not sure why this is needed to fix bash completions, but it's a bit procedural for my likes.

Think you can refactor like this

```bash
linked_completions=(
  lxc-attach lxc-cgroup lxc-console lxc-destroy lxc-device lxc-execute
  lxc-freeze lxc-info lxc-monitor lxc-snapshot lxc-stop lxc-unfreeze
)

(
  cd $out/share/bash-completion/completions

  mv lxc lxc-start
  
  for completion in ''${linked_completions[@]}; do
    ln -sfn lxc-start $completion
  done
)

```

Reply:
Updated as requested.